lib/cpu.rb

```diff
@@ -2,15 +2,15 @@ class Cpu
     def initialize(board)
         @board = board
     end
-    def cpus_turn
+
+    def take_turn
         @board.grid.each_with_index do |row, row_index|
             row.each_with_index do |cell, column_index|
                 if !cell.painted?
-                    @board.paint!(row_index,column_index)
-                    @board.agent_is_cpu
+                    @board.paint!(row_index, column_index, :cpu)
                     return
                 end
             end
         end
     end
-end
\ No newline at end of file
+end
```

Comment:
When you added this file it looks like your editor used 4 spaces per tab; usual style and the rest of the project is 2.

Reply:
I've been manually typing each space when I do two. I liked the 4 space tab in my other project and didn't update my settings for that reason. But, it looks like I can update this setting on a per-project basis according to Google, so there's no reason not to update that. Sorry for the inconsistency 😄